core/server/src/main/java/alluxio/master/file/FileSystemMaster.java

```diff
@@ -1544,6 +1544,20 @@ public void mount(AlluxioURI alluxioPath, AlluxioURI ufsPath)
     MasterContext.getMasterSource().incMountOps(1);
     synchronized (mInodeTree) {
       checkPermission(FileSystemAction.WRITE, alluxioPath, true);
+
+      // Check that the Alluxio Path does not exist
+      boolean pathExists = false;
+      try {
+        mInodeTree.getInodeByPath(alluxioPath);
```

Comment:
Other than using exception handling for the logic control, shall we add a method checking the existence?